Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -0,0 +1,19 @@
+function targetBlank() {
+    const links = document.querySelectorAll('a');
+    const currentHostname = window.location.hostname;
+
+    for (let i = 0; i < links.length; i++) {
+        const link = links[i];
+        const linkHostname = link.hostname;
+
+        if (linkHostname !== currentHostname && link.href && !link.href.startsWith('#') && !link.getAttribute('rel') && !link.target) {
```

Comment:
Why did you rewrite the conditions?

- I don't think that you need `!link.href.startsWith('#')` since, checking the link hostname also checks that.
- Why do you need to check `rel` and `target` (It was not on the original script)?
- You are not checking for `mailto` and for `javascript:` (the original script was checking for that), so target blank gets added to them:
![image](https://user-images.githubusercontent.com/80963259/219696700-53b34dc9-eda8-4eba-babb-cebfcf606b63.png)
 (using the original script it's not added to these `<a>` tags).